### ISSUE TITLE
nfd: missing Depends systemd

### ISSUE THREAD

**Issue by yoursunny** [named-data/ppa-packaging#16]
`nfd-start` command installed by `nfd` package uses `systemctl` command which comes from `systemd` package, so `systemd` should be listed as a dependency.

**Comment by cawka** [comment]
No. Such dependency should not be specified. The custom-content `nfd-start`/`stop` scripts was my attempt to prevent starting NFD by non-system provided means.  If I figure out how to not install these script, I would do it instead.  If it is too complicated, then I'll just print a diagnostic message that other way of starting NFD should be used.

**Comment by yoursunny** [comment]
I could agree that a dependency on *systemd* is not desired.
However, as long as it's still needed by the current script, it should be specified as a dependency.
